README.md

```diff
@@ -69,63 +69,63 @@ API
 `ENDING` is either `.html` or `.json`.
 
 - General
-  - `GET /`  
+  - `GET /`
     Show a description of the project.
     - Link to this repository#readme.
     - Link to download the code /source.
     - How to contribute.
     - What this is about.
-  - `GET /source`  
+  - `GET /source`
     Get the source code as a zip file.
-- Github Authentication:
+- GitHub Authentication:
   - `GET /auth`
     Show a form to register a new authentication.
-  - `POST /auth`  
-    Add `username` and `password` to those usable to scrape github.
+  - `POST /auth`
+    Add `username` and `password` to those usable to scrape GitHub.
     They will be tried and removed if invalid.
 - Organization
-  - `GET /organizations<ENDING>`  
+  - `GET /organizations<ENDING>`
     [schema][organizations-schema]
-    [example responses][organizations-examples]  
-    List all organizations with links to their statuses.    
+    [example responses][organizations-examples]
+    List all organizations with links to their statuses.
     Users may be listed among them.
     Arguments: `offset=0&limit=10`
-    - offset is the number of organizaions to skip
+    - offset is the number of organizations to skip
     - limit is the number of organizations to display
-  - `GET /organization/<organization><ENDING>`  
+  - `GET /organization/<organization><ENDING>`
     [schema][organization-schema]
-    [example responses][organization-examples]  
+    [example responses][organization-examples]
     Get an organization and its status.
     Same as the element of `/organizations.json`
     Ending:
     - `json`
     - `html`
     - `svg` ![](https://img.shields.io/badge/First%20Timers-1-blue.svg)
-  - `POST /organization<ENDING>`  
+  - `POST /organization<ENDING>`
     Submit an `organization` for scraping.
     This shows an html page with a link to the status of the organization.
 - Repository
-  - `GET /repositories<ENDING>`  
```

Comment:
Two spaces at the end are a newline in markdown. Would you like to remove the newline?